docs: show how to use optional options in the schema example (#706)
express-validator/express-validator: docs/feature-schema-validation.md

  },
  // Wildcards/dots for nested fields work as well
  'addresses.*.postalCode': {

>     // Make this field optional and allow it to be null

Reviewer: ```suggestion
    // Make this field optional when undefined or null
```
